app/src/main/java/com/orange/ouds/app/ui/tokens/TokenPropertyHeader.kt

```diff
@@ -170,6 +218,12 @@ private fun PreviewGridHeader() = OudsPreview {
     GridHeader()
 }
 
+@UiModePreviews.Default
+@Composable
+private fun PreviewSizeIconWithTextHeader() = OudsPreview {
+    SizeIconWithTextHeader(size = OudsSizeKeyToken.Icon.WithHeading.Small.SizeLarge.value, tokenName = "Heading.Small.SizeLarge")
```

Comment:
@florentmaitre I don't know why the style of the text is not applied in the preview, if you find the reason...

Reply:
The text style is not applied because the `getTokens` method that we use to find the appropriate text style uses reflection, which does not work in AS previews. This can be fixed this way:

```kotlin
val style = if (LocalInspectionMode.current) {
    OudsTypographyKeyToken.Heading.Small.value
} else {
    OudsTypographyKeyToken::class.getTokens()
        .asOrNull<List<Token<TextStyle>>>()
        ?.firstOrNull { typographyToken ->
            typographyToken.name.replace(".Strong", "") == tokenTypography
        }
        ?.value?.invoke()
        .orElse { LocalTextStyle.current }
}
```